src/System.Linq/ref/System.Linq.builds

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Linq.csproj" />
+    <Project Include="System.Linq.csproj">
+      <TargetGroup>netcoreapp1.1</TargetGroup>
```

Comment:
Can you please flip and make netcoreapp1.1 the default and the netstandard16 the secondary configuration?